lib/discord_bot/commands/slash/nice_message.rb

```diff
@@ -0,0 +1,34 @@
+module DiscordBot
+  module Commands
+    module Slash
+      class NiceMessage < Base
+        class << self
+          def description
+            "Provides the source code for #{Config.bot_name}"
+          end
+          def register
+            Bot.register_command(command_name, description) do |command|
+              command.user(:user, 'What user to message?', required: true)
+              command.string(:message, 'what message to send', required: true)
+            end
+          end
+          
+          def run(command)
+            user=DiscorBot::User.new(user: Bot.find_user_by_id(command.options["user"]))
```

Comment:
You reverted the commit that fixed this typo. You should undo that revert commit.

Reply:
fixed as well